data/metric/source_type.go

```diff
@@ -13,26 +13,41 @@ type SourceType int
 // Source types
 // If any more SourceTypes are added update maps: SourcesTypeToName & SourcesNameToType.
 const (
-	// GAUGE is a value that may increase and decrease. It is stored as-is.
+	// GAUGE is a value that may increase and decrease.
+	// It generally represents the value for something at a particular moment in time
 	GAUGE SourceType = iota
```

Comment:
shouldn't we use CamelCase?

Reply:
it was already in ALL_CAPS so i left it. It's not go "friendly", I can change that but for now we just ignoring the warning